client/src/rpc_client.rs

```diff
@@ -48,6 +49,17 @@ pub struct RpcClient {
     sender: Box<dyn RpcSender + Send + Sync + 'static>,
 }
 
+fn serialize_encode_transaction(
+    transaction: &Transaction,
+    encoding: RpcTransactionEncoding,
+) -> String {
+    let serialized = serialize(transaction).unwrap();
+    match encoding {
+        RpcTransactionEncoding::Base58 => bs58::encode(serialized).into_string(),
+        RpcTransactionEncoding::Base64 => base64::encode(serialized),
```

Comment:
Happy to only support base64 actually, and never let a Rust client send slomo base58

Reply:
+1

Reply:
Yep!  I think we need to roll that out over a couple releases to maintain our initiative to "break less stuff".  I'm planning to:
1. Add support for base64 here, maintaining base58 defaults and announcing base58 deprecation.
1. Drop base58 client-side in the next release, reiterating deprecation
1. Drop base58 server-side in the release after that, by now everyone's had fair warning.

I'll toss up some issues to track if you agree

Reply:
👍

Reply:
Migration tracking issues at #12700 and #12701